### PR TITLE
add Backlog tab select columns docs

### PR DESCRIPTION
**Pull request by peterVG**

### CODE REVIEW

File: user-manual/backlog/backlog.rst [hunk 1]
```diff
@@ -46,6 +46,44 @@ Searching for transfers and files
       :width: 80%
       :alt: Archivematica backlog tab showing all transfers in the backlog
 
+#. To change the columns that are visible in the tabular list of backlogged transfers,
+   click on the Select columns icon at the bottom of the table.
```
Comment (sallain) [inline user-manual/backlog/backlog.rst]:
```suggestion
   click on the **Select columns** icon at the bottom of the table.
```

File: user-manual/backlog/backlog.rst [hunk 2]
```diff
@@ -46,6 +46,44 @@ Searching for transfers and files
...
+
+#. This will present a list of the columns that are available for viewing in the
```
Comment (sallain) [inline user-manual/backlog/backlog.rst]: I would combine this point with the previous one, as they're really the same step.

File: user-manual/backlog/backlog.rst [hunk 3]
```diff
@@ -46,6 +46,44 @@ Searching for transfers and files
...
+
+   .. figure:: images/backlog-select-columns.*
+      :align: center
+      :width: 80%
+      :alt: Archivematica backlog tab showing select columns options
+
+#. Click on a blue or white row to toggle this value for each column. In the
+   example below, all the available columns are selected. Note that your column
+   selection choices will persist between your user sessions and apply to other
+   users of the same Archivematica pipeline.
```
Comment (sallain) [inline user-manual/backlog/backlog.rst]:
```suggestion
   users of the same Archivematica instance.
```

**Review by sallain — CHANGES_REQUESTED** [review]
Just a couple of suggestions!

File: user-manual/backlog/backlog.rst (hunk 2 above)
Comment (peterVG) [reply user-manual/backlog/backlog.rst]: Good point. Done.

**Review by sevein — APPROVED** [review]

**Review by sallain — APPROVED** [review]

**Pull request merged**